src/stdpar/plugin-PixelTrackFitting/PixelTrackSoAFromCUDA.cc

```diff
@@ -1,42 +1,23 @@
-#include <cuda_runtime.h>
-
-#include "CUDACore/Product.h"
 #include "CUDADataFormats/PixelTrack.h"
 #include "Framework/EventSetup.h"
 #include "Framework/Event.h"
 #include "Framework/PluginFactory.h"
 #include "Framework/EDProducer.h"
-#include "CUDACore/ScopedContext.h"
 
-class PixelTrackSoAFromCUDA : public edm::EDProducerExternalWork {
+class PixelTrackSoAFromCUDA : public edm::EDProducer {
```

Comment:
In principle this module is now useless (e.g. the simple mock framework can't handle multiple modules producing the same type of product). In `cudauvm` it played to role of issuing prefetches to the host memory, but in `stdpar` even that does not make much sense. I suppose for the "transfer to host" tests we need to measure the time of something actually accessing all the memory (e.g. `HistoValidator`).